cli/src/main.rs

```diff
@@ -283,40 +314,18 @@ fn build_app<'a, 'b>() -> App<'a, 'b> {
                         SubCommand::with_name("all")
                             .about("Fetch all trades")
                             .arg(
-                                Arg::with_name("base_asset_type")
-                                    .long("base_asset_type")
-                                    .takes_value(true)
-                                    .help("Filters trades with a base_asset_type"),
-                            )
-                            .arg(
-                                Arg::with_name("base_asset_code")
-                                    .long("base_asset_code")
-                                    .takes_value(true)
-                                    .help("Filters trades with a base_asset_code.  Not required for XLM"),
-                            )
-                            .arg(
-                                Arg::with_name("base_asset_issuer")
-                                    .long("base_asset_issuer")
+                                Arg::with_name("base_asset")
+                                    .long("base_asset")
                                     .takes_value(true)
-                                    .help("Filters trades with a base_asset_issuer.  Not required for XLM"),
-                            )
-                            .arg(
-                                Arg::with_name("counter_asset_type")
-                                    .long("counter_asset_type")
-                                    .takes_value(true)
-                                    .help("Filters trades with a counter_asset_type"),
-                            )
-                            .arg(
-                                Arg::with_name("counter_asset_code")
-                                    .long("counter_asset_code")
-                                    .takes_value(true)
-                                    .help("Filters trades with a counter_asset_code.  Not required for XLM"),
+                                    .required(true)
+                                    .help("Filters trades with a given base_asset. format:  <asset_code>-<asset_issuer>, or xlm if lumens"),
                             )
                             .arg(
-                                Arg::with_name("counter_asset_issuer")
-                                    .long("counter_asset_issuer")
+                                Arg::with_name("counter_asset")
+                                    .long("counter_asset")
```

Comment:
Do we really need to type `_asset` everytime we use this? I feel like just `base` and `counter` might be slightly easier to use on the command line.